client/components/Mentions.vue

```diff
@@ -171,6 +200,11 @@ export default {
 		messageTime(time) {
 			return dayjs(time).fromNow();
 		},
+		removeMentions() {
```

Comment:
This certainly would be better to fire off as a single event to the server. Something like `mentions:clear_all` would be simple, then you can clear `state.mentions` once too.